src/utils/scrubber.coffee

```diff
@@ -48,6 +48,8 @@
 
           v = that.getClosestPoint(series.values, axes.xScale.invert(x))
 
+          dispatch.focus(v, series.values.indexOf(v))
```

Comment:
We could probably support clicks on the glass in scrubber mode, with some sort of indication passed along the data about the distance from data points...